Misc/NEWS.d/next/Tests/2023-04-25-23-56-04.gh-issue-gh-82054.3GgB66.rst

```diff
@@ -0,0 +1 @@
+Porting parallel test execution from cinder by sharding test execution. Note that this is being done only on the asyncio and compiler tests since they are executed serially.
```

Comment:
We can remove this since the change isn't end-user-facing.

Reply:
Ok I will fix these.

Reply:
I have resolved this. See 
https://github.com/python/cpython/pull/103859/commits/f4b476b4419ccbf53ce50382265458cb3237f25d